pkgs/tools/misc/findup/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, zig, testVersion, findup }:
+
+stdenv.mkDerivation rec {
+  pname = "findup";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "hiljusti";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-erlKIiYYlWnhoeD3FnKdxnHjfGmmJVXk44DUja5Unig=";
+  };
+
+  nativeBuildInputs = [ zig ];
+
+  # Builds and installs (at the same time) with Zig.
+  dontConfigure = true;
+  dontBuild = true;
+
+  # Give Zig a directory for intermediate work.
+  preInstall = ''
+    export HOME=$TMPDIR
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    zig build -Drelease-safe -Dcpu=baseline --prefix $out
+    runHook postInstall
+  '';
+
+  passthru.tests.version = testVersion { package = findup; };
```

Comment:
aliases (in general) are not allowed in nixpkgs. `testVersion` is an alias of `testers.testVersion`.

so change it to testers.testVersion.

Reply:
I see, thank you!